frappe/desk/doctype/setup_wizard_settings/setup_wizard_settings.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019, Frappe Technologies and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe.model.document import Document
+from frappe.modules import get_module_path, scrub_dt_dn
+from frappe.modules.export_file import export_to_files, create_init_py
+
+class SetupWizardSettings(Document):
+	def on_update(self):
+		if frappe.flags.in_import or frappe.flags.in_test:
+			return
+
+		if frappe.local.conf.get('developer_mode'):
+			record_list = [['Setup Wizard Settings', self.name]]
+
+			for s in self.slide_order:
+				record_list.append(['Setup Wizard Slide', s.slide])
+
+			export_to_files(record_list=record_list, record_module='Desk')
```

Comment:
Does this mean only slides added to the slide order child table will be exported?

Reply:
Yes

Reply:
added slide_order in Slide Doctype and removed Setup Wizard Settings.